src/pages/home/sidebar/SidebarView.js

```diff
@@ -31,6 +38,7 @@ const SidebarView = props => (
         {!props.isChatSwitcherActive && (
             <SidebarBottom insets={props.insets} />
         )}
+        <FAB animatedValue={props.fabAnimatedValue} onPress={props.fabOnPress} isHidden={false} />
```

Comment:
**Note:** `isHidden` must be set to `true` before we merge.

Reply:
Noticed that after the push. Just stuck a note on my monitor to not forget again!